### PR TITLE
Update rejuvenate.lua

### PR DESCRIPTION
**Pull request by Ariphaos**
Adds an -age parameter and works more flexibly with old_age. Should probably be merged into gm-unit but this was faster for me to edit. I wanted my dwarves to be older than 20 is all.

### CODE REVIEW

File: rejuvenate.lua [hunk 1]
```diff
@@ -1,43 +1,47 @@
...
+Sets the age of the selected dwarf to 20 years old by default. Useful if valuable citizens
 are getting old.
 
 Arguments:
 
 - ``-all``: applies to all citizens
-- ``-force``: also applies to units under 20 years old. Useful if there are too many babies around...
+- ``-force``: also applies to units under 20 (or the age specified with -age) years old. Useful if there are too many babies around...
 - ``-dry-run``: only list units that would be changed; don't actually change ages
+- ``-age``: set to the specified age, rather than 20.
```
Comment (myk002) [inline rejuvenate.lua:17]: this doc section is actually leftover from a doc migration we did last year. Could you delete this doc string and add your doc updates to `docs/rejuvenate.rst`?

**Review by myk002 — COMMENTED** [review]
looks great! Thanks!

File: rejuvenate.lua (hunk 1 above)
Comment (Ariphaos) [reply rejuvenate.lua:17]: To confirm you want me to remove all documentation from the script?

Comment (Ariphaos) [reply rejuvenate.lua]: Donezo.

File: docs/rejuvenate.rst [hunk 2]
```diff
@@ -2,11 +2,12 @@ rejuvenate
 ==========
 
 .. dfhack-tool::
-    :summary: Sets unit age to 20 years.
+    :summary: Sets unit age to 20 years by default.
```
Comment (myk002) [inline docs/rejuvenate.rst:5]:
```suggestion
    :summary: Resets unit age.
```

File: docs/rejuvenate.rst [hunk 3]
```diff
@@ -24,14 +25,18 @@ Examples
     Set the age of all dwarves over 20 to 20.
 ``rejuvenate --all --force``
     Set the age of all dwarves (including babies) to 20.
+ ``rejuvenate --age 149 --force``
```
Comment (myk002) [inline docs/rejuvenate.rst:28]:
```suggestion
``rejuvenate --age 149 --force``
```

File: docs/rejuvenate.rst [hunk 4]
```diff
@@ -24,14 +25,18 @@ Examples
...
+    Set the age of the selected dwarf to 149, even if they are younger.
 
 Options
 -------
 
 ``--all``
     Rejuvenate all citizens, not just the selected one.
+``--age``
```
Comment (myk002) [inline docs/rejuvenate.rst:36]:
```suggestion
``--age <num>``
```

**Comment by myk002** [comment]
Could you also add a changelog.txt entry for this?

**Review by myk002 — APPROVED** [review]

**Pull request merged**